receiver/splunkhecreceiver/config.go

```diff
@@ -60,17 +60,10 @@ func (c *Config) initialize() error {
 	if c.HostKey == "" {
 		c.HostKey = conventions.AttributeHostName
 	}
-
-	path := c.Path
-	if path == "" {
-		path = "*"
-	}
-	glob, err := glob.Compile(path)
-	if err != nil {
-		return err
+	if c.RawPath == "" {
```

Comment:
Should we also log a warning if `path` is provided?

Reply:
done